test/e2e/utils/chaise.page.js

```diff
@@ -314,6 +314,16 @@ var recordEditPage = function() {
         return browser.executeScript("return $('td.entity-value textarea[name=\"" + name + "\"]')[" + index + "];");
     };
 
+    this.getJSONtextAreaForAcolumn = function(name, index) {
+       index = index || 0;
+       return browser.executeScript("return $('td.entity-value textarea[name=\"" + name + "\"]')[" + index + "];");
+   };
+   
+   this.getJSONBtextAreaForAcolumn = function(name, index) {
+       index = index || 0;
+       return browser.executeScript("return $('td.entity-value textarea[name=\"" + name + "\"]')[" + index + "];");
+   };
```

Comment:
And these are not different from `getTextAreaForAcolumn` function. You could just reuse that function.